tests/ssl-opt.sh

```diff
@@ -8807,8 +8807,8 @@ run_test    "DTLS reordering: Buffer out-of-order hs msg before reassembling nex
             hs_timeout=2500-60000" \
             0 \
             -c "Buffering HS message" \
-            -c "attempt to make space by freeing buffered future messages" \
-            -c "Enough space available after freeing buffered HS messages" \
+            -C "attempt to make space by freeing buffered future messages" \
```

Comment:
Please explain in the commit message why you have to update those error messages.

Reply:
Done